### PR TITLE
Grahampheath: telophase fix and keymaps

### PR DESCRIPTION
**Pull request by loadedsith**
Fixes an issue with programming my Let's Split clone - "The Divide". Adds 2 more emoji to my custom keymaps.

### CODE REVIEW

File: keyboards/telophase/telophase.h [hunk 1]
```diff
@@ -62,6 +62,7 @@
 	{ k36, k37, k38, k39, k40, k41,      k42, k43, k44, k45, k46, k47 }, \
 }
 
+#define FLIP_HALF
```
Comment (drashna) [inline keyboards/telophase/telophase.h]: Is there a reason that this is added here?

Comment (loadedsith) [reply keyboards/telophase/telophase.h]: I see, it doesn't do anything over here. I was under the impression this was required. Thanks!

**Comment by loadedsith** [comment]
Removed the include ../../config and the flip_halves lines.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**